### PR TITLE
docs: adding sms readme

### PR DESCRIPTION
**Pull request by manchuck**
## Description
adding sms readme

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/sms/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+You can also use this SDK standalone if you only need access to just the
+SMS API.
+
+### With NPM
+
+```bash
+npm install @vonage/sms
+```
+
+### With Yarn
```
Comment (dragonmantank) [inline packages/sms/README.md:40]: We should either add yarn to each Readme, or remove it from here

File: packages/sms/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Using the Vonage SMS SDK
+
+### As part of the Vonage Server SDK
+
+If you are using this SDK as part of the Vonage Server SDK, you can access it
+as the `sms` property off of the client that you instantiate.
+
+```js
+const { Vonage, Auth } = require('@vonage/server-sdk');
```
Comment (dragonmantank) [inline packages/sms/README.md]: We don't expose Auth here

**Review by dragonmantank — CHANGES_REQUESTED** [review]

File: packages/sms/README.md (hunk 1 above)
Comment (manchuck) [reply packages/sms/README.md:40]: We should have it in each readme. Is there one that is missing?

Comment (dragonmantank) [reply packages/sms/README.md:40]: For some reason it only stood out here us having it. I don't know why.

**Review by dragonmantank — APPROVED** [review]

**Pull request merged**